README.md

```diff
@@ -0,0 +1,34 @@
+# Wicked Migration
+
+Provides a webserver for migrating Wicked configuration files to NetworkManager configuration files.
+
+### Build:
+
+`cargo build --release`
+
+### Run Server:
+
+Options
+| Option | Description |
+|--------|-------------|
+| -s     | Specify where static files are located |
+| -i     | Set IP address |
+| -p     | Set port |
+| -d     | Set location for the database |
+| -h     | Show help information |
+
+Configuration files are automatically deleted after 5 minutes or upon retrieval.
```

Comment:
I think it would make sense to mention somewhere that configurations are stored in the configured temp directory and also that podman is required to be installed.